### DISCUSSION TITLE
Disabling cache auto update

### DISCUSSION THREAD

**Discussion by CaptainFrosty** [tldr-pages/tldr-node-client#379] [Q&A] [answered]
Is there a way to disable cache auto update in the node.js client? It is almost always typos that cause it, and it updates very slowly on my remote server for some reason.

**Comment by sbrl** [comment]
Hey there! I'm transferring your discussion to the repository for the Node.js client for tldr-pages. This repository is just for the pages themselves, not the clients.

**Comment by kbdharun** [comment, marked as answer]
> Is there a way to disable cache auto update in the node.js client? It is almost always typos that cause it, and it updates very slowly on my remote server for some reason.

Sorry for the late reply, you can indeed disable auto-updating when a page is not found by updating the [configuration](https://github.com/tldr-pages/tldr-node-client#configuration) file and setting this variable `skipUpdateWhenPageNotFound` to true.
